src/backend/columnar/cstore_tableam.c

```diff
@@ -743,7 +743,8 @@ LogRelationStats(Relation rel, int elevel)
 	appendStringInfo(infoBuf,
 					 "total row count: %ld, stripe count: %d, "
 					 "average rows per stripe: %ld\n",
-					 tupleCount, stripeCount, tupleCount / stripeCount);
+					 tupleCount, stripeCount,
+					 stripeCount ? tupleCount / stripeCount : 0);
```

Comment:
guess this as the division by 0 that caused the error? Makes sense to have this change. I was suspecting this to be the only change. Some more information on the PR, but maybe more so adding examples on the comments on the other changes, would help understand the change set better.